cmd/kube-controller-manager/app/core_test.go

```diff
@@ -107,7 +126,15 @@ func TestStartResourceQuotaController_DiscoveryError(t *testing.T) {
 			InformerFactory:  informers.NewSharedInformerFactoryWithOptions(testClientset, time.Duration(1)),
 			InformersStarted: make(chan struct{}),
 		}
-		_, _, err := startResourceQuotaController(ctx)
+		var err error
```

Comment:
I don't think we need to declare the error here as we are not concerned with persisting the value.

Reply:
Ooops. Removed